akm_tools/validation/validator.py

```diff
@@ -50,9 +50,12 @@ def _configure_data_instance_validators(self):
         ]
         return data_instance_validator_objects
 
-    def log_errors(self):
+    def log_errors(self,silent_mode):
         for error_msg in self.validation_errors:
             logging.error(error_msg)
+        if len (self.validation_errors)>0:
```

Comment:
if not(silent_mode):

More pythonic to be positive: if(silent_mode) or maybe if(notify_mode)

Reply:
fixed.